api/kv.go

```diff
@@ -50,21 +50,21 @@ type KVOp string
 
 const (
 	KVSet          KVOp = "set"
-	KVDelete            = "delete"
-	KVDeleteCAS         = "delete-cas"
-	KVDeleteTree        = "delete-tree"
-	KVCAS               = "cas"
-	KVLock              = "lock"
-	KVUnlock            = "unlock"
-	KVGet               = "get"
-	KVGetTree           = "get-tree"
-	KVCheckSession      = "check-session"
-	KVCheckIndex        = "check-index"
+	KVDelete       KVOp = "delete"
```

Comment:
Morning @slackpad. Sorry to make such a fuzz about a few lines, but it's not the same thing. The types don't get copied over; it happens to work because the definitions without a type are untyped string constants. Not specifying a type at all also works fine:

```
type KVOp string

const (
    KVSet          = "set"
    KVDelete       = "delete"
...
```

If you only add a KVOp type for `KVSet` then `KVSet` will be a typed string, but `KVDelete` will still be untyped. So I would either add the type on every line, or not at all.

Have a look at my Gist posted above, and reread the 'String constants' part from https://blog.golang.org/constants .